app/src/main/res/layout/item_comment_product.xml

```diff
@@ -12,20 +12,19 @@
     <android.support.v7.widget.CardView
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/dp_3"
-        android:padding="@dimen/dp_3"
+        android:layout_margin="@dimen/dp_1"
+        android:padding="@dimen/dp_2"
         >
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_margin="@dimen/dp_5"
-            android:layout_marginTop="@dimen/dp_10"
             android:orientation="vertical"
+            android:padding="@dimen/dp_2"
+            android:paddingRight="@dimen/dp_2"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingRight` you should probably also define `paddingLeft` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.